.travis.yml

```diff
@@ -0,0 +1,27 @@
+# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
+# For more info for the package, see https://github.com/ros-industrial/industrial_ci/blob/master/README.rst
+sudo: required 
+dist: trusty 
+services:
+  - docker
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    recipients:
+      - noobaca2@gmail.com
```

Comment:
This was not written anywhere or mentioned anywhere previous to this comment @ibaranov-cp. I've since amended the Wiki to include this tradition.